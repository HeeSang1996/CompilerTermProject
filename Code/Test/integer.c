void main() {
    int a = 0;
    int b = 1;
    int c = 1234567890987654321;
    int d = -1;
    int f = -1234567890987654321;
}