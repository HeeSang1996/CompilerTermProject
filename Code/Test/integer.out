ID void
ID main
PAREN (
PAREN )
BRACE {
VARIABLE int
ID a
ASSIGN =
INT 0
TERM ;
VARIABLE int
ID b
ASSIGN =
INT 1
TERM ;
VARIABLE int
ID c
ASSIGN =
INT 1234567890987654321
TERM ;
VARIABLE int
ID d
ASSIGN =
INT -1
TERM ;
VARIABLE int
ID f
ASSIGN =
INT -1234567890987654321
TERM ;
VARIABLE int
ID g
ASSIGN =
INT 0
TERM ;
VARIABLE int
ID h
ASSIGN =
INT 1
TERM ;
VARIABLE int
ID i
ASSIGN =
INT 1234567890987654321
TERM ;
VARIABLE int
ID j
ASSIGN =
INT -1
TERM ;
VARIABLE int
ID k
ASSIGN =
INT -1234567890987654321
TERM ;
VARIABLE int
ID l
ASSIGN =
INT 0
TERM ;
VARIABLE int
ID m
ASSIGN =
INT 1
TERM ;
VARIABLE int
ID n
ASSIGN =
INT 1234567890987654321
TERM ;
VARIABLE int
ID o
ASSIGN =
INT -1
TERM ;
VARIABLE int
ID p
ASSIGN =
INT -1234567890987654321
TERM ;
BRACE }
